Pod/Classes/ios/NYTPhotosViewController.h

```diff
@@ -207,6 +207,11 @@ extern NSString * const NYTPhotosViewControllerDidDismissNotification;
  */
 - (void)photosViewController:(NYTPhotosViewController *)photosViewController actionCompletedWithActivityType:(NSString * _Nullable)activityType;
 
+/**
+ *  Dismisses `NYTPhotosViewController`. Should only be used in cases where user interaction of a button in the captionView requires a different viewController to be modal.
+ */
+- (void)forceDismiss;
```

Comment:
I thought about having that too, wasn't sure because of the delegate handling didDismiss and if they would both fire or not.